code/datums/traits/negative.dm

```diff
@@ -1,5 +1,87 @@
 //predominantly negative traits
 
+/datum/trait/depression
+	name = "Depression"
+	desc = "You sometimes just hate life."
+	mob_trait = TRAIT_DEPRESSION
+	value = -1
+	gain_text = "<span class='danger'>You start feeling depressed.</span>"
+	lose_text = "<span class='notice'>You no longer feel depressed.</span>" //if only it were that easy!
+	medical_record_text = "Patient has a severe mood disorder causing them to experience sudden moments of sadness."
+
+
+
+/datum/trait/family_heirloom
+	name = "Family Heirloom"
+	desc = "You are the current owner of an heirloom. passed down for generations. You have to keep it safe!"
+	value = -1
```

Comment:
Minus points for what is essentialy free mood unless you suck?

Reply:
You have to clutter your inventory to lug around a random object that, in likelihood, won't have any use.

Reply:
just have it be 0 value to be safe

Reply:
I view the concept as inherently negative, so I might end up removing the mood boost instead.

Reply:
it has a passive buff, but I suppose its fair for the 1 mood boost and it means this is an attractive negative trait pick, which I like. So fine with me :D

Reply:
agreed